apps/kyb-app/src/components/organisms/DynamicUI/rule-engines/json-schema.rule-engine.ts

```diff
@@ -74,7 +75,10 @@ export class JsonSchemaRuleEngine implements RuleEngine {
       return fieldErrors;
     });
 
-    return result?.flat()?.filter(result => Boolean(result.message));
+    return uniqBy(
+      result?.flat()?.filter(result => Boolean(result.message)),
+      'message',
+    );
```

Comment:
Isn't a message connected to a field? It seems the real issue is that more than one of the same message is being pushed to the same field.

Reply:
Duplicate messages are coming from ajv, the way we parsing errors and converting them to field errors didnt handled this case. So uniqBy should be fine.

Reply:
If 3 fields have the same message, would only the first field have the message?

Reply:
Why in the first place we get couple of messages on a single input field? @chesterkmr